data/hotel.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Created a new file for hotel as appending an address book json and a hotel json together in 1 file using method here: [https://github.com/AY1920S2-CS2103T-W13-2/main/pull/85/files#r398621381](url)  some json style errors.